dist/js/apps/events-page/app.js

```diff
@@ -158,7 +158,7 @@ bcpl.boostrapCollapseHelper = function ($) {
 	var app = angular.module('events', []);
 
 	var constants = {
-		baseUrl: 'https://services.bcpl.info',
+		baseUrl: 'https://testservices.bcpl.info',
```

Comment:
Should this have changed?

Reply:
no, good catch, i thought i removed that.

Reply:
@ascire fixed